### PR TITLE
Fixing rent node error

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Fixing rent node error
### Changes

-Fixing toast error while insufficient balance
### Related Issues

#1804
### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by AlaaElattar** [comment]
- It kept loading and nothing happened 

[Screencast from 12-31-2023 01:24:20 PM.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/3c7c1a15-4753-4758-984b-db1c987d0886)

**Comment by AlaaElattar** [comment]
> * It kept loading and nothing happened
> 
>  Screencast.from.12-31-2023.01.24.20.PM.webm

But this contract in grace period, is this the reason whey it didn't get deleted ?

**Review by AlaaElattar — APPROVED** [review]
- works fine

image

File: packages/playground/src/dashboard/components/reserve_action_btn.vue [hunk 1]
```diff
@@ -98,8 +99,9 @@ export default {
           }, 20000);
         }
       } catch (e) {
-        console.log("Error: ", e);
-        createCustomToast(e as string, ToastType.danger);
+        if (e instanceof InsufficientBalanceError) {
+          createCustomToast(`Can't delete rent contract due to Insufficient balance`, ToastType.danger);
+        }
```
Comment (0oM4R) [inline packages/playground/src/dashboard/components/reserve_action_btn.vue:104]: we should have else here

File: packages/playground/src/dashboard/components/reserve_action_btn.vue [hunk 2]
```diff
@@ -121,7 +126,7 @@ export default {
         }, 20000);
       } catch (e) {
         console.log("Error: ", e);
-        createCustomToast(e as string, ToastType.danger);
+        createCustomToast(normalizeError(e, "Failed to rent node."), ToastType.danger);
```
Comment (0oM4R) [inline packages/playground/src/dashboard/components/reserve_action_btn.vue:129]: image

i tried to remove the @checkBalance decorator to get  chain error and its not the wanted error msg, we can make it more clear to the user

**Review by 0oM4R — APPROVED** [review]
image

**Pull request merged**